### ISSUE TITLE
[Bug] the due date is showing in decimals 

### ISSUE THREAD

**Issue by kulizivo** [j-palindrome/obsidian-time-ruler#135]
![image](https://github.com/user-attachments/assets/5174f095-3947-4ae2-9354-af1887f4c175)

This is how the date is displayed

**Comment by floroh** [comment]
Duplicate of #133 , but still an issue in current version 2.5.2

**Comment by kulizivo** [comment]
@floroh yeah didnt see it till I already posted this one, since the issue was already closed
